doc/tutorials/enable_ivshmem.rst

```diff
@@ -59,21 +60,23 @@ where
 
 -  ``shm_size`` - Shared memory size of selected ``shm_name``.
 
-There are two ways to insert above boot parameter for ``acrn-dm``
+There are two ways to insert the above boot parameter for ``acrn-dm``:
 
--  Manually edit launch script file, in this case, user shall ensure that both
-   ``shm_name`` and ``shm_size`` match with that are defined via configuration tool GUI.
+-  Manually edit the launch script file. In this case, ensure that both
```

Comment:
We should check to see if this launch script option can be set in the configurator too (vs. manually editing the script).